apps/www/app/layout.tsx

```diff
@@ -0,0 +1,12 @@
+export const metadata = {
+  title: 'Next.js',
+  description: 'Generated by Next.js',
```

Comment:
Reminder to change this from the defaults, though do we even need this file? @fsansalvadore

Reply:
yeah was thinking if we can just remove this actually - these were all generated as part of the Sentry's set up wizard